apps/web/src/app/(private)/settings/_components/ProviderApiKeys/New/index.tsx

```diff
@@ -51,44 +55,81 @@ export default function NewApiKey({
     >
       <form id='createApiKeyForm' action={action}>
         <FormWrapper>
-          <FormField label='Provider'>
-            <Select
-              required
-              name='provider'
-              onChange={(newValue) => setProviderType(newValue)}
-              defaultValue={data?.provider}
-              options={PROVIDER_OPTIONS}
-            />
-          </FormField>
-          <FormField label='Name'>
+          <FormField
+            label='Provider'
+            description="This is the name you'll use in the prompt editor to refer to use this provider and model."
+          >
```

Comment:
No need for `<FormField>`. Remove it and pass `label` and `description` to input.

Reply:
Okay, will change, although it was already like this, I only added another field.

Reply:
I know. I just noticed this bad pattern in another place
https://github.com/latitude-dev/latitude-llm/pull/578

So the idea here is to be a good citizen and adapt the code now that you're touching it